src/plotlydash.py

```diff
@@ -0,0 +1,103 @@
+import dash
+import dash_core_components as dcc
+import dash_html_components as html
+import plotly.express as px
+import redis
+from dash.dependencies import Input, Output
+
+external_stylesheets = ['https://codepen.io/chriddyp/pen/bWLwgP.css']
+
+app = dash.Dash(__name__, external_stylesheets=external_stylesheets)
+
+file = 'colorCount.csv'
```

Comment:
The code from here to the `@app.callback` (line 70 currently) should be in a `setup` function.  Keep `app` global so you can reference it.